polytracker/database.py

```diff
@@ -942,7 +961,7 @@ def parent_two(self) -> Optional["DBTaintForestNode"]:
         return self._parent_two
 
     @staticmethod
-    def taints(labels: Iterable["DBTaintForestNode"]) -> Taints:
+    def get_taints(labels: Iterable["DBTaintForestNode"]) -> Taints:
```

Comment:
I believe this should be reflected in the examples in README.md

Reply:
Ugh, it's complicated. This change should not affect the public API, because a `taints` function still exists in the superclass, so the examples in the docs should still work. I renamed the static version to `get_taints` in order to avoid the name conflict with the version used in the documentation that is not static.